apps/demos/Demos/HtmlEditor/MarkdownSupport/jQuery/index.html

```diff
@@ -8,11 +8,26 @@
     <script src="../../../../node_modules/jquery/dist/jquery.min.js"></script>
     <link rel="stylesheet" type="text/css" href="../../../../node_modules/devextreme-dist/css/dx.light.css" />
     <script src="../../../../node_modules/devextreme-quill/dist/dx-quill.min.js"></script>
-    <script src="../../../../node_modules/turndown/dist/turndown.js"></script>
-    <script src="../../../../node_modules/showdown/dist/showdown.js"></script>
+
+    <script type="module">
+      import { unified } from "https://esm.sh/unified@11?bundle";
+      import remarkParse from "https://esm.sh/remark-parse@11?bundle";
+      import remarkRehype from "https://esm.sh/remark-rehype@11?bundle";
+      import rehypeStringify from "https://esm.sh/rehype-stringify@10?bundle";
+      import rehypeParse from "https://esm.sh/rehype-parse@9?bundle";
+      import rehypeRemark from "https://esm.sh/rehype-remark@10?bundle";
+      import remarkStringify from "https://esm.sh/remark-stringify@11?bundle";
+
+      window.unified = unified;
+      window.remarkParse = remarkParse;
+      window.remarkRehype = remarkRehype;
+      window.rehypeStringify = rehypeStringify;
+      window.rehypeParse = rehypeParse;
+      window.rehypeRemark = rehypeRemark;
+      window.remarkStringify = remarkStringify;
```

Comment:
cannot we get rid of `window.* =` how u did in BC sample?

Reply:
It depends on which file we want to initialise the converter in. We can create it in script with tag ‘module’ in index.html, add it to window as window.converter. Then use it in index.js to pass in options. But that doesn't seem like a clear option to me.